english/find-exit/into-mountains/mountains.md

```diff
@@ -10,4 +10,6 @@ Do you:
 
 [Continue upwards?](./upwards/upwards.md)
 
+[Push boulders into the ravine.](./boulder/boulder.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.